crv/crvShape.cc

```diff
@@ -300,66 +304,65 @@ class EdgeReshaper : public ma::Operator
   int nr;
 };
 
-static bool isCornerTriAngleLarge(crv::Adapt *a,
+static bool isCornerTriAngleLargeMetric(crv::Adapt *a,
     ma::Entity* tri, int index)
 {
   ma::Mesh* m = a->mesh;
-  apf::Element* elem = apf::createElement(m->getCoordinateField(),tri);
-  apf::NewArray<apf::Vector3> nodes;
-  apf::getVectorNodes(elem,nodes);
-  apf::destroyElement(elem);
+  // get downward vertexes
+  ma::Entity* down[3];
```

Comment:
not necessary. This happens inside a static function which is only called from other functions the including .cc file. Something else would fail much earlier before this call.